Functions/QualityControl/xASL_qc_CollectParameters.m

```diff
@@ -34,37 +35,43 @@
     error('Missing iSubject...');
 end
 
+% It is recommended to provide session number for ASL
+if nargin<4 || isempty(iSession)
+	iSession = 1;
+	if strcmp(ScanType, 'ASL')
```

Comment:
Not only ASL

Reply:
Taking all together - agree. Will do, don't resolve yet.

Reply:
Done in 0b7a798c

Reply:
We said that we would put all sessions to 1 right, irregardless of the scantype? So I would remove the scantype checking here, and just give a general warning. Or perhaps only checking for non-structural.

Reply:
Yes, that's more or less the case. 
1) We put all sessions to 1 - that is in the code already (in case that the session is missing). So that should be fine.
2) For structural, we don't really use session, so the checking is indeed not needed.

So I propose that I give a general warning, remove IF and print the warning using the `ScanType` variable. And we check if that is structural, we don't print the warning. Agree?

Reply:
OK agree. will do!